cli/index.js

```diff
@@ -54,17 +53,16 @@ const outputMode = cli.flags.output || 'pretty';
 const outputPath = cli.flags.outputPath || 'stdout';
 const flags = cli.flags;
 
-const driver = new ChromeProtocol();
-
 // set logging preferences
+flags.logLevel = 'info';
```

Comment:
it makes sense for the module to default to `quiet`, a bunch of console noise from a dependency is annoying, but the *current* default is `info` and it may make sense for the CLI to stay there (@paulirish can weigh in) for the time being. So maybe default `flags.logLevel` here to `info`?